gather.go

```diff
@@ -9,7 +9,7 @@ import (
 	"net"
 	"time"
 
-	"github.com/danderson/nat/stun"
+	"github.com/chripell/nat/stun"
```

Comment:
Good point, especially because I would like to add some bigger features soon. Will close this pull request and make a more appropriate one.